src/test/java/uk/gov/register/resources/RegisterCommandWriterTest.java

```diff
@@ -93,7 +97,10 @@ public void writeTo_shouldAddContentDispositionHeader() throws IOException {
         }).when(httpHeadersMock).add(eq("Content-Disposition"), anyObject());
 
         ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
-        RegisterCommandWriter sutCommandWriter = new RegisterCommandWriter();
+        CanonicalJsonMapper canonicalJsonMapper = new CanonicalJsonMapper();
+        CanonicalJsonValidator canonicalJsonValidator = new CanonicalJsonValidator();
+        CommandParser commandParser = new CommandParser(canonicalJsonMapper, canonicalJsonValidator);
+        RegisterCommandWriter sutCommandWriter = new RegisterCommandWriter(commandParser);
```

Comment:
It might be nice to extract the above lines into a method called `getRegisterCommandWriter()` and extract the code so that we don't duplicate so much.